### DISCUSSION TITLE
"Predefined routes take precedence over dynamic routes" - NextJS Docs

### DISCUSSION THREAD

**Discussion by nath-modules** [vercel/next.js#28559] [Help]
They do. Yet when building the project I get

path: **"/somepath"** from page: **"/[slug]"** conflicts with path: **"/somepath"** 

error - Conflicting paths returned from getStaticPaths, paths must unique per page.

Am I doing something wrong here? if not I'll raise an issue.

docs: [https://nextjs.org/docs/routing/dynamic-routes#caveats](url)

**Comment by wwwdepot** [comment]
You cannot have duplicate paths. Choose one or the other.

**Reply by nath-modules** [reply]
nice of them to add a caveat to waste my time. Thanks for the reply
